### ISSUE TITLE
Training doesn't seem to converge and there are no bounding boxes (only with low threshold and these are wrong)

### ISSUE THREAD

**Issue by NielsDebrier** [thtrieu/darkflow#649]
Hello everyone

I am trying to train my own model on my own dataset. But that didn't go as planned so now I am trying  with images and annotations of the PASCAL VOC set and even that doesn't work. If someone can help me and tell me what I am overlooking or forgetting, that would be really appreciated.

As suggested in other issues I tried with a small dataset to overfit the data.
I took 15 images from the PASCAL VOC with cars in and their corresponding annotation file.
Then I removed all objects that where not cars from the annotation file.

As stated in the read.me I changed the classes to 1 and filter to 30.
And in the labels.txt I only wrote Car.

Command used: 
`./flow --model cfg/yolo-car.cfg --load bin/yolo.weights --train --annotation annotation/ --dataset dataset/ --gpu 0.90 --batch 8`

__Training statistics:__
- Dataset size: 15
- Learning rate : 1e-05
- Batch size    : 8
- Epoch number  : 1000
- Backup every  : 2000

__Last output:__
> step 1000 - loss 0.156395465136 - moving ave loss 0.157443178127
> Checkpoint at step 1000
> Finish 1000 epoch(es)
> Training finished, exit.

Looking at the output it didn't seemed converged (still going lower each time)
So I let it run a second time:
`./flow --model cfg/yolo-car.cfg --load -1 --train --annotation annotation/ --dataset dataset/ --gpu 0.90 --batch 8`

__Last output:__
> step 2000 - loss 4.58002868982e-05 - moving ave loss 4.66413631906e-05
Checkpoint at step 2000
Finish 1000 epoch(es)
Training finished, exit.

Loss is still seems to be going down. If we try to run the model against the dataset it doesn't show boxes only when we lower the threshold quite a bit.
`time ./flow --model cfg/yolo-car.cfg --load 2000 --imgdir dataCar/ --gpu 1.0 --threshold 0.001`
and the boxes that are visible then are wrong (for example a lot in the sky on 2 images and none on the cars.)

So if someone has suggestions on what to try next or on what to look into that would be appreciated. 
Also I quite new to deep learning and training a model so every advice or tips for that matter are welcome.

For those who want here is a google link to the files I used: [google drive](https://drive.google.com/drive/folders/1GGBrVyaKiY9uTINAKJtQ5ySDPSPKAkGw.)
Last commit from darkflow: 5fe906b347ab8584354e375a14ea2e7bdf5188d7

**Comment by zsfarkas** [comment]
+1
